site/utils/apiUtils/index.ts

```diff
@@ -81,3 +90,8 @@ export const getPageState = <T>(errorCookie: string, schemaObject: z.ZodType<T>,
 
     return inputsProps;
 };
+
+export const getReturnPage = (req: NextApiRequest) => {
```

Comment:
Should this function ever be returning null? It's called getReturnPage but can return null. Should we not just be throwing an error if we cant get the return page?

Reply:
When there are no query params then the function shouldn't interrupt the page. An error thrown would have to be caught in the api and logged

Reply:
Yep fair enough